embeddings/evaluator/sequence_labeling_evaluator.py

```diff
@@ -65,6 +65,24 @@ def metrics(
     ) -> Sequence[Metric[Union[List[Any], nptyping.NDArray[Any], torch.Tensor], Dict[Any, Any]]]:
         return [self.metric]
 
+    @classmethod
+    def get_metric_name(
+        cls, evaluation_mode: EvaluationMode, tagging_scheme: Optional[TaggingScheme] = None
+    ) -> str:
```

Comment:
Why is it `classmethod`? `cls` is not used, 'staticmethod` would be more intuitive.